docs/integrating-scrapy.md

```diff
@@ -86,6 +86,14 @@ Install Python dependencies using the provided requirements file named `requirem
 pip install -r requirements-apify.txt [-r requirements.txt]
 ```
 
+Finally execute the Apify Actor.
+
+```bash showLineNumbers
+apify run [--purge]
+```
+
+If [ActorDatasetPushPipeline](https://github.com/apify/apify-sdk-python/blob/master/src/apify/scrapy/pipelines.py) is configured, the output of the Actor should be located on the location `storage/datasets/default/`.
```

Comment:
This sentence seems needlessly convoluted , wouldn't something like this be better 

```suggestion
If [ActorDatasetPushPipeline](https://github.com/apify/apify-sdk-python/blob/master/src/apify/scrapy/pipelines.py) is configured, the Actor's output will be stored in the `storage/datasets/default/` directory.
```